src/js/_enqueues/wp/emoji.js

```diff
@@ -277,16 +277,7 @@
 			return twemoji.parse( object, params );
 		}
 
-		/**
-		 * Initialize our emoji support, and set up listeners.
-		 */
-		if ( settings ) {
-			if ( settings.DOMReady ) {
```

Comment:
And since `wp-emoji-loader.js` now runs as a script module, it is deferred to execute at the point that the DOM is constructed, so we know it is ready to go.